org.opent2t.sample.thermostat.superpopular/test/thermostatTests.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+var OpenT2T = require('opent2t').OpenT2T;
+
+function runThermostatTests(translatorPath, mockDevice, test, testData) {
```

Comment:
investigate being able to use this same file unmodified for testing against devices